### PR TITLE
[DOCS-7597]Alfresco Office Services (AOS) 3.0

### PR DESCRIPTION
**Pull request by MateuszTomaska**

### CODE REVIEW

File: google-drive/latest/support/index.md
Comment (MateuszTomaska) [inline google-drive/latest/support/index.md:1]: This is for https://alfresco.atlassian.net/browse/DOCS-7581 and is a mistake on my part in the placement in this ticket.

Comment (anxumalo) [reply google-drive/latest/support/index.md:1]: Easily fixed - let's discuss :-)

File: microsoft-office/2.0/admin/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+```text
+Blocked a directory listing request from MS-Office. This indicates a broken MS-Office
+deployment. Please check that the ROOT and the _vti_bin webapps are deployed properly and
+reachable from the outside!
+```
+
+To fix this problem, ensure that you have installed the Alfresco Office Services AMP file, and deployed the `_vti_bin.war` file that is required for AOS to work correctly.
+
+See [Install Alfresco Office Services]({% link microsoft-office/latest/install/index.md %}) for more.
```
Comment (anxumalo) [inline microsoft-office/2.0/admin/index.md:33]: This should link to `microsoft-office/2.0/...` - check for similar links in this **2.0** folder

**Review by anxumalo — CHANGES_REQUESTED** [review]
- [x] Add commit for update to `microsoft-office/latest/support/index.md`
- [x] Check links in `microsoft-office/2.0/` remain in `microsoft-office/2.0/` folder (only 1 found to `/latest/`)
- [x] Revert update to `google-drive/latest/support/index.md`

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**